parser/shared/src/main/scala/hmda/parser/fi/panel/PanelCsvParser.scala

```diff
@@ -0,0 +1,33 @@
+package hmda.parser.fi.panel
+
+import javax.swing.JFileChooser
+import javax.swing.filechooser.FileNameExtensionFilter
+
+import scala.io.Source
+import scala.util.Try
+
+object PanelCsvParser extends App {
+  val file = pickFile()
+  val lines = file.getLines().toList.tail
+
+  for (line <- lines) {
+    val i = InstitutionParser(line)
+    println(i)
+  }
+
+  def pickFile(): Source = {
+    val chooser = new JFileChooser()
```

Comment:
I think this file needs to be moved to the `panelJVM` project. It contains `Java Swing` desktop components and file access methods that are `JVM` only, and while we will never run this from the `panelJS` if we attempted to do that (i.e. by mistake) it would fail.